bazel/prebuilt.bzl

```diff
@@ -0,0 +1,732 @@
+        deps = [
+            "lightstep_includes",
+            "protobuf",
+        ],
+    )
+
+    native.cc_library(
+        name = "lightstep_includes",
+        hdrs = LIGHTSTEP_INCLUDES_HDRS,
+        includes = [
+            "thirdparty_build/lightstep.dep/include",
+        ],
+    )
+
+    native.cc_library(
+        name = "nghttp2",
+        srcs = NGHTTP2_LIBS,
+        hdrs = NGHTTP2_HDRS,
+        includes = [
+            "thirdparty_build/nghttp2.dep/include",
+        ],
+    )
+
+    native.cc_library(
+        name = "protobuf",
+        srcs = PROTOBUF_LIBS,
+        hdrs = PROTOBUF_HDRS,
+        includes = [
+            "thirdparty_build/protobuf.dep/include",
+        ],
+    )
+
+    native.cc_library(
+        name = "rapidjson",
+        hdrs = RAPIDJSON_HDRS,
+        includes = [
+            "thirdparty/rapidjson.dep/rapidjson-1.1.0/include",
+        ],
+    )
+
+    native.cc_library(
+        name = "spdlog",
+        hdrs = SPDLOG_HDRS,
+        includes = [
+            "thirdparty/spdlog.dep/spdlog-0.11.0/include",
+        ],
+    )
+
+    native.cc_library(
+        name = "ssl",
+        srcs = SSL_LIBS,
+        deps = [
+            "crypto",
+        ],
+    )
+
+    native.cc_library(
+        name = "tclap",
+        hdrs = TCLAP_HDRS,
+        includes = [
+            "thirdparty/tclap.dep/tclap-1.2.1/include",
+        ],
+    )
+
+    native.filegroup(
+        name = "protoc",
+        srcs = PROTOC_LIBS,
+    )
+
+def envoy_prebuilt_workspace_targets(path):
+    native.bind(
+        name = "ares",
+        actual = path + ":ares",
+    )
+
+    native.bind(
+        name = "crypto",
+        actual = path + ":crypto",
+    )
+
+    native.bind(
+        name = "event",
+        actual = path + ":event",
+    )
+
+    native.bind(
+        name = "event_pthreads",
+        actual = path + ":event_pthreads",
+    )
+
+    native.bind(
+        name = "googletest",
+        actual = path + ":googletest",
+    )
+
+    native.bind(
+        name = "http_parser",
+        actual = path + ":http_parser",
+    )
+
+    native.bind(
+        name = "lightstep",
+        actual = path + ":lightstep",
+    )
+
+    native.bind(
+        name = "lightstep_includes",
+        actual = path + ":lightstep_includes",
+    )
+
+    native.bind(
+        name = "nghttp2",
+        actual = path + ":nghttp2",
+    )
+
+    native.bind(
+        name = "protobuf",
+        actual = path + ":protobuf",
+    )
+
+    native.bind(
+        name = "rapidjson",
+        actual = path + ":rapidjson",
+    )
+
+    native.bind(
+        name = "spdlog",
+        actual = path + ":spdlog",
+    )
+
+    native.bind(
+        name = "ssl",
+        actual = path + ":ssl",
+    )
+
+    native.bind(
+        name = "tclap",
+        actual = path + ":tclap",
+    )
+
+    native.bind(
+        name = "protoc",
+        actual = path + ":protoc",
+    )
+
+    # Used only for protobuf.bzl
+    native.git_repository(
```

Comment:
we had similar hack in istio before, if you only intended use `protobuf.bzl` from this protobuf repo, use `new_git_repository` with `build_file_content = ""`, it will disable the BUILD file in protobuf root, so the anything else (cc_library, etc) won't be loaded.